crates/web-plugins/didcomm-messaging/protocols/basic-message/src/handler.rs

```diff
@@ -0,0 +1,39 @@
+use crate::error::ProtocolError;
+use crate::model::{BasicMessage, MessageBody};
+use axum::http::StatusCode;
+use axum::response::{IntoResponse, Response};
+use chrono::Utc;
+use uuid::Uuid;
+
+pub fn basic_message(content: &str, lang: Option<String>) -> Result<BasicMessage, ProtocolError> {
```

Comment:
* All functions are asynchronous. Please refer on the other implementations.
* Provide a better name to this handler. See the naming convention of other handlers.
* The signature of the handler is not consistent with the signature used for all the other protocols.